torchio/transforms/preprocessing/spatial/resample.py

```diff
@@ -74,41 +70,21 @@ def __init__(
             ):
         super().__init__(**kwargs)
         self.target = target
-        self.reference_image, self.target_spacing = self.parse_target(target)
-        parsed_interpolation = self.parse_interpolation(image_interpolation)
-        self.image_interpolation = parsed_interpolation
+        self.image_interpolation = self.parse_interpolation(
+            image_interpolation)
         self.pre_affine_name = pre_affine_name
         self.scalars_only = scalars_only
+        self.target_shape = None
+        self.target_affine = None
```

Comment:
not used ?

Reply:
You're right, good catch! I was trying a different design and this is a trace of that.